bundle/config/root.go

```diff
@@ -11,8 +11,31 @@ import (
 	"github.com/imdario/mergo"
 )
 
-// FileName is the name of bundle configuration file.
-const FileName = "bundle.yml"
+type ConfigFileNames [4]string
+
+// FileNames is the name of bundle configuration file.
+var FileNames = ConfigFileNames{"databricks.yaml", "databricks.yml", "bundle.yaml", "bundle.yml"}
+
+func (c ConfigFileNames) FindInPath(path string) (string, error) {
+	result := ""
+	var err error = nil
+
+	for _, file := range c {
+		filePath := filepath.Join(path, file)
+		_, err = os.Stat(filePath)
+		if err == nil {
+			if result != "" {
+				return "", fmt.Errorf("multiple bundle configuration files found in %s", path)
+			}
+			result = filePath
+		}
+	}
+
+	if result == "" {
+		return "", err
```

Comment:
This would return the last error. It would be good to return only the first error, so the message includes the preferred bundle root configuration file name.

Reply:
Good catch. Fixed that